### PR TITLE
OHRM5X-1131: Initiate translator for back-end translations

### PR DESCRIPTION
**Pull request by RajithaKumara**

### CODE REVIEW

File: symfony/plugins/orangehrmLeavePlugin/Report/LeaveTypeLeaveEntitlementUsageReport.php [hunk 1]
```diff
@@ -69,7 +71,7 @@ public function getHeaderDefinition(): Header
     {
         return new Header(
             [
-                (new Column(self::PARAMETER_EMPLOYEE_NAME))->setName('Employee')
+                (new Column(self::PARAMETER_EMPLOYEE_NAME))->setName($this->getI18NHelper()->trans('general.employee'))
                     ->setPin(Column::PIN_COL_START)
                     ->setSize(self::DEFAULT_COLUMN_SIZE),
                 (new Column(self::PARAMETER_ENTITLEMENT_DAYS))->setName('Leave Entitlements (Days)')
```
Comment (SharukaNP) [inline symfony/plugins/orangehrmLeavePlugin/Report/LeaveTypeLeaveEntitlementUsageReport.php:77]: I assume we can translate the others later right? like Leave Entitlements (Days)

**Review by SharukaNP — COMMENTED** [review]
Looks good to me. I think I might get an idea from the collection you created for translations when improving the tool I made

File: symfony/plugins/orangehrmLeavePlugin/Report/LeaveTypeLeaveEntitlementUsageReport.php (hunk 1 above)
Comment (RajithaKumara) [reply symfony/plugins/orangehrmLeavePlugin/Report/LeaveTypeLeaveEntitlementUsageReport.php:77]: yes

File: symfony/plugins/orangehrmI18NPlugin/Service/I18NService.php [hunk 2]
```diff
@@ -143,4 +266,71 @@ public function getETagByLangCode(string $langCode): string
...
+    }
+
+    /**
+     * @param string $key
+     * @param array $parameters
+     * @param string|null $langCode
+     * @return string
+     * @internal
+     */
+    public function trans(string $key, array $parameters = [], string $langCode = null): string
```
[the remark this answers is not in the document]
Comment (RajithaKumara) [reply symfony/plugins/orangehrmI18NPlugin/Service/I18NService.php:300]: use same name as https://symfony.com/doc/current/translation.html#basic-translation

**Pull request merged**